dist/utils.js

```diff
@@ -423,6 +417,8 @@ function instanceOf(obj) {
     return false;
   }
 
+  if (arguments.length === 2 && arguments[1] === 'object') return arguments[0] && !instanceOf(arguments[0], 'string', 'number', 'boolean', 'array', 'function');
```

Comment:
'object' type is now handled differently. If 'object' is requested, the check is: `object is truthy and object is not a string, number, boolean, array, or function`